src/Cli/Microsoft.TemplateEngine.Cli/TemplatePackageDisplay.cs

```diff
@@ -156,7 +156,7 @@ internal void DisplayBuiltInPackagesCheckResult(string packageId, string version
                               string.Format(
                                   LocalizableStrings.TemplatePackageCoordinator_lnstall_Error_AlreadyInstalled,
                                   packageToInstall).Bold().Red());
-                        _reporterError.WriteLine(LocalizableStrings.TemplatePackageCoordinator_lnstall_Error_AlreadyInstalled_Hint, InstallCommand.ForceOption.Aliases.First());
+                        _reporterError.WriteLine(LocalizableStrings.TemplatePackageCoordinator_lnstall_Error_AlreadyInstalled_Hint, InstallCommand.ForceOption.Name);
```

Comment:
i'm a bit hesitant here - in examples we need to show alias, and not name.
this applies to other cases.

Reply:
just to confirm my understanding, after this change, the `Aliases` collection has only 2nd and so on aliases, and name will be always a "primary" one?

Reply:
Yep! Here's what CliOption looks like, [for example](https://github.com/dotnet/command-line-api/blob/main/src/System.CommandLine/CliOption.cs#L19-L29).